### ISSUE TITLE
Could not find generator 'config:install'

### ISSUE THREAD

**Issue by Mirigan** [railsconfig/config#156]
Hi, 
I just install the gem and when I try to use "rails g config:install" I get the message 

> Could not find generator 'config:install'

I'm working with rails 5 and the gem's version 1.3.0

**Comment by pkuczynski** [comment]
@Mirigan why did you closed it? Is the problem solved?

**Comment by Mirigan** [comment]
Yes I don't know why but when I retry this morning the command worked

**Comment by pkuczynski** [comment]
I see. Interesting...

**Comment by dyodia** [comment]
I had same issue. But I shut down the terminal and re-open it, then it's solved. I have no idea about the reason but I hope it helps someone.
